### PR TITLE
feat: avatar added

### PR DESCRIPTION
**Pull request by Tanjum786**
Screenshot from 2022-02-02 04-33-37
Screenshot from 2022-02-02 04-39-36

### CODE REVIEW

File: component/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+.avatar_xs_size {
+    border: 1px solid black;
+    border-radius: 50%;
+    font-size: 1rem;
+    height: 3rem;
+    width: 3rem;
+
+}
+
```
Comment (krishanu2911) [inline component/avatar/avatar.css:67]: As in the above avatar variations classes border, border-radius.. were repeating so you can make a common class with all the similar properties

File: component/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
+.avatar_heading {
+    display: flex;
+    flex-wrap: wrap;
+    color: var(--third-color);
```
Comment (tanishq20) [inline component/avatar/avatar.css]: Give the generic name to colors. So it will be easy to recall and reuse.

File: component/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+.avatar_ex {
+    display: flex;
+    flex-wrap: wrap;
+    color: var(--third-color);
+    font-size: 2rem;
+    margin-left: 2rem;
+    margin-top: 2rem;
+}
+
+.para {
```
Comment (tanishq20) [inline component/avatar/avatar.css]: I'm supposing that you are using para in all the doc pages. If that is the case then you can shift this to a utility class.

File: component/avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    width: 6rem;
+}
+
+.avatar_xs_size {
+    border: 1px solid black;
+    border-radius: 50%;
+    font-size: 1rem;
+    height: 3rem;
+    width: 3rem;
+
```
Comment (tanishq20) [inline component/avatar/avatar.css]: Avoid giving these extra space.

File: component/avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    justify-content: center;
+    background-color: var(--primary-color);
+    color: white;
+}
+
+.primary-text-color {
+    color: var(--primary-color);
+}
+
+.avatarCode {
```
Comment (tanishq20) [inline component/avatar/avatar.css]: Follow one convention for all the class names. According to your structure, this should be avatar_code.

File: component/avatar/avatar.css (hunk 1 above)
Comment (Tanjum786) [reply component/avatar/avatar.css:67]: okay  I will do that

File: component/avatar/avatar.css (hunk 3 above)
Comment (Tanjum786) [reply component/avatar/avatar.css]: Okay I will shift :)

File: component/avatar/avatar.css (hunk 4 above)
Comment (Tanjum786) [reply component/avatar/avatar.css]: Thank You.Noted

**Review by tanishq20 — COMMENTED** [review]
Good to merge

**Pull request merged**